swap/swap.go

```diff
@@ -54,14 +53,14 @@ const swapLogLevel = 3 // swapLogLevel indicates filter level of log messages
 // A node maintains an individual balance with every peer
 // Only messages which have a price will be accounted for
 type Swap struct {
-	store            state.Store        // store is needed in order to keep balances and cheques across sessions
-	peers            map[enode.ID]*Peer // map of all swap Peers
-	peersLock        sync.RWMutex       // lock for peers map
-	backend          contract.Backend   // the backend (blockchain) used
-	owner            *Owner             // contract access
-	params           *Params            // economic and operational parameters
-	contract         swap.Contract      // reference to the smart contract
-	honeyPriceOracle HoneyOracle        // oracle which resolves the price of honey (in Wei)
+	store            state.Store           // store is needed in order to keep balances and cheques across sessions
+	peers            map[enode.ID]*Peer    // map of all swap Peers
+	peersLock        sync.RWMutex          // lock for peers map
+	backend          swapcontract.Backend  // the backend (blockchain) used
+	owner            *Owner                // contract access
+	params           *Params               // economic and operational parameters
+	contract         swapcontract.Contract // reference to the smart contract
```

Comment:
i would also change this field here (as well as the comment). 

this seems to be the chequebook, not just any smart contract.

Reply:
(just in case: i meant the `contract` field here)